### DISCUSSION TITLE
What kind of tree can you carry in your hand 🖐️?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1144] [Galaxy Trivia] [answered]
**Bonus:** If ten 🐦🐦 are sitting in a tree and a hunter shoots one, how many birds are left in the tree?

**Comment by thuongtruong1009** [comment, marked as answer]
You can carry a palm tree in your hand 🖐️.

If a hunter shoots one of the ten birds sitting in a tree, the loud noise and commotion would likely scare the remaining birds away. So, there may be no birds left in the tree after the shot.
